client/lib/plans/constants.js

```diff
@@ -1180,6 +1243,24 @@ export const FEATURES_LIST = {
 			'Also includes SEO previews and Google Analytics.'
 		),
 		hideInfoPopover: true
+	},
+
+	[ FEATURE_CONCIERGE_SETUP ]: {
+		getSlug: () => FEATURE_CONCIERGE_SETUP,
+		getTitle: () => i18n.translate( 'Concierge Setup' ),
+		getDescription: () => i18n.translate(
+			'Schedule time with one of our Happiness Engineers.'
+		),
+		hideInfoPopover: true
+	},
+
+	[ FEATURE_MARKETING_AUTOMATION ]: {
+		getSlug: () => FEATURE_MARKETING_AUTOMATION,
+		getTitle: () => i18n.translate( 'Marketing Automation' ),
+		getDescription: () => i18n.translate(
+			'Schedule all your social media posts in advance.'
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'You can now schedule your social media posts in advance with Jetpack Premium and Professional.' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).